Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[workspace]
+resolver = "2"
+members = ["formatting"]
+
+[workspace.package]
+version = "0.1.0"
```

Comment:
Should the version go into the crate-specific `Cargo.toml`?

Reply:
Doesn't need to, at least for now. Only once we have more crates, if it turns out we need to release them separately. For now it's a bit easier to keep as much as possible in the root `Cargo.toml` I think, but I'm fine either way.